### ISSUE TITLE
Rake task fails with composite tag value option

### ISSUE THREAD

**Issue by sashaCher** [cucumber/cucumber-ruby#1614]
**Describe the bug**

When cucumber rake task `cucumber_opts` field is a string and it contains composite tag option like `--tags 'not @skip'` cucumber rake task fails with `No such file or directory - @skip'. You can use `cucumber --init` to get started.` error.

**To Reproduce**
1. Create new cucumber rake task like this
   ```ruby
   desc 'Run features'
   Cucumber::Rake::Task.new(:features) do |t|
     t.cucumber_opts = "--tag 'not @skip'"
   end
   task :cucumber => :features
   ```
   Where `cucumber_opts` task property is set to a string value ends by composite tag option, `` for example
1. Run the rake task `bundle exec rake cucumber`
1. See error: ``No such file or directory - @skip'. You can use `cucumber --init` ``

**Expected behavior**

The rake task is running scenarios meets chosen tags criteria.
In the above example all scenarios except having `@skip` tag will run.

**Additional context**

Looks like the next split is a root cause of the issue: https://github.com/cucumber/cucumber-ruby/blob/3ebc762c3b9717e8593cd28f55ad356e287ebef6/lib/cucumber/rake/task.rb#L116

**Comment by valerianb** [comment]
The preferred type is an array.

It should work fine if you pass:
```ruby
t.cucumber_opts = ["--tag", "'not @skip'"]
```

**Comment by sashaCher** [comment]
Thank you @valerianb!
It's definitely worked.
I guess more strict approach to parameters validation/warning that preferred type is an array was able to save me couple hours of frustration after cucumber gem upgrade.

**Comment by sashaCher** [comment]
I guess that a warning that string type `cucumber_opts` does not support all options and it's recommended to use array type will improve troubleshooting around the code area.
Exactly like warning about `~` replacement by `not` was/is.
I've found also another example in the repo supports this approach https://github.com/cucumber/cucumber-ruby/blob/9777e60cef0b97421f54c5ae2ff49b838baef917/lib/cucumber/formatter/unicode.rb#L20

In my case I upgraded cucumber to version 7 and then `~` negation started throw an exception. Good, actually, exception with explanation what I need to do: to replace it by `not`. But replacing by `not` threw other exception, that it was quite hard to figure out what's a root cause of it.
